build.gradle

```diff
@@ -32,16 +32,20 @@ repositories {
 }
 
 dependencies {
-    compileApi ('org.apache.jclouds:jclouds-core:2.0.1') 
-    compileApi ('com.google.auto.service:auto-service:1.0-rc3') 
+    compile ('com.google.code.gson:gson:2.5') {
+        force = true
+    }
+    compileApi ('org.apache.jclouds:jclouds-core:2.0.1')
+    compileApi ('com.google.auto.service:auto-service:1.0-rc3')
     compileApi ('com.google.auto.value:auto-value:1.4.1')
-    
+    compileApi ('com.ryanharter.auto.value:auto-value-gson:0.4.5')
```

Comment:
It's not last version because jcloud don't support gson 2.8 (https://issues.apache.org/jira/browse/JCLOUDS-1160)